Kitodo/src/main/webapp/WEB-INF/templates/includes/taskTemplateEdit/taskEditDetails.xhtml

```diff
@@ -18,7 +18,7 @@
         xmlns:ui="http://xmlns.jcp.org/jsf/facelets">
     <p:panelGrid columns="2" layout="grid">
         <p:row>
-            <p:repeat var="generator" value="#{TemplateForm.task.generators}">
```

Comment:
According to design this page is going to be removed in very near future. Probably it is not the page to which you want to add it? What about taskEdit/taskEditDetails?

Reply:
According to what I have agreed with Elena, the additional switches should be added to the existing switches at tasks template. I had followed it on the edge that you have removed the other functions here. This has unfortunately overlapped. These changes should have been done before, if the branch would have been merged in mid-August, since then it's around. So basically these switches have to go to the others too. But that's what I would suggest in a separate ticket, otherwise it gets too confusing.

Reply:
But this is wrong page... It should be tasks of process. I will remove this page in next week.

Reply:
No, that's not it. It can not be the task of the process, otherwise you would have to configure it individually for each process. It just needs to be configured just once, such as metadata step, script step, etc. The switches you have moved away. These switches must also be moved away if you want to remove the page. But one step at a time. I also find it awkward that this has overlapped in time now. But we have to live with that. In any case, it will need its own ticket, because there really has to be a lot of change. If I have seen it correctly, the configuration for metadata step, script step etc. are now in the workflow. If the configuration for the content to be generated and validated must be there, this means that the folder configuration must also be moved to the workflow. Basically, when I think about it, I'm absolutely in favor of it. The workflow also has a lot to do with the folders with content, they are finally processed in the workflow. Basically, they have always been an alien in the project because they have nothing to do with the project. But as I said, that's going to be a big change. Let's make our own ticket for it.